src/Components/Search/NewSearch/NewSearchInputPillsContainer.tsx

```diff
@@ -0,0 +1,203 @@
+import {
+  Box,
+  Flex,
+  Pill,
+  ShelfNavigationProps,
+  ShelfNext,
+  ShelfPrevious,
+} from "@artsy/palette"
+import { useEffect, useRef, useState } from "react"
+import styled, { css } from "styled-components"
+
+const PILLS_OPTIONS = [
+  "TOP",
+  "ARTWORK",
+  "ARTIST",
+  "ARTICLE",
+  "SALE",
+  "ARTIST_SERIES",
+  "COLLECTION",
+  "FAIR",
+  "SHOW",
+  "GALLERY",
+]
+
+const ChevronStyle = css`
+  position: absolute;
+  width: 30px;
+  height: 30px;
+  @media (hover: none) {
+    display: none;
+  }
+`
+
+const PreviousChevron = styled(ShelfPrevious)<ShelfNavigationProps>`
+  ${ChevronStyle}
+`
+
+const NextChevron = styled(ShelfNext)<ShelfNavigationProps>`
+  ${ChevronStyle}
+`
+
+interface GradientBgProps {
+  placement: "right" | "left"
+}
+
+const GradientBg = styled(Box)<GradientBgProps>`
+  position: "absolute";
+  right: 0;
+  width: 100px;
+  height: 30px;
+  background: transparent;
+  background-image: linear-gradient(
+    to ${props => props.placement},
+    rgba(255, 255, 255, 0) 0%,
+    rgba(255, 255, 255, 1) 40%
+  );
+`
+
+const PillsContainer = styled(Flex)`
+  flex-wrap: nowrap;
+  overflow-x: auto;
+  scrollbar-width: none;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+`
+
+const GRADIENT_BG_WIDTH = 30
+
+export const NewSearchInputPillsContainer = () => {
```

Comment:
Lets be really diligent about refactoring out / deleting the "old" search when this launches, so that there's not components with "New" in the title all about. This problem is quite chronic in Eigen so want to be sure it doesn't spread to Force!  

Another approach would be to rename existing search to `LegacySearch` _right now_, and then this component, rather than being called `NewSearch`, can be renamed to `Search`. If this work isn't a spike then this feels like a safe approach.